### ISSUE TITLE
Untagged orphan VPCs created during AWS provisioning

### ISSUE THREAD

**Issue by abutcher** [openshift/openshift-ansible#8029]
#### Description

The `openshift_aws` role creates untagged orphan VPCs due to an issue with `ec2_vpc_net` module not waiting long enough for VPCs to exist before attempting to tag them.

We believe this is fixed by https://github.com/ansible/ansible/pull/37354 available within the devel branch of ansible upstream (2.6).

##### Version

```
openshift-ansible-3.10.0-0.13.0-363-g4340e3dab
ansible-2.5.0-2.fc27.noarch
```

**Comment by jctanner** [comment]
https://github.com/ansible/ansible/pull/38975

**Comment by ryansb** [comment]
https://github.com/ansible/ansible/pull/38975 has been merged, but missed the 2.5.1 window. It will be available in Ansible 2.5.2.
